plugins/module_utils/redfish_utils.py

```diff
@@ -912,7 +912,26 @@ def get_volume_inventory(self, systems_uri):
                             else:
                                 sc_id = sc[0].get('Id', '1')
                                 controller_name = 'Controller %s' % sc_id
+                    if 'Controllers' in data:
```

Comment:
I would make this an `elif`; if `StorageControllers` was found, there's no need to step into `Controllers`. Both will represent the same storage controllers in the storage subsystem.

Reply:
I see your point. Changing as suggested.